.buildkite/publish-ruby-gems.sh

```diff
@@ -51,15 +51,28 @@ with_backoff() {
 # sorbet-static, but the sorbet-static gem push failed.
 #
 # (By failure here, we mean that RubyGems.org 502'd for some reason.)
-for gem_archive in "_out_/gems/sorbet-static-$release_version"-*.gem; do
-  if ! gem list --remote rubygems.org --exact 'sorbet-static' | grep -q "$release_version"; then
-    with_backoff gem push --verbose "$gem_archive"
+# Push the linux gem
+if ! gem fetch sorbet-static --platform x86_64-linux --version "$release_version" | grep -q "ERROR"; then
```

Comment:
nice... this is a better fix than what we did in https://github.com/sorbet/sorbet/pull/2283

Reply:
thanks!